config/config.php

```diff
@@ -2,11 +2,14 @@
 
 return [
     'auth' => [
-        'enabled' => true,
+        'enabled' => env('FLIGHTDECK_AUTH_ENABLED', true)
     ],
     'tokens' => [
         'expire_days' => 30
     ],
+    'cors' => [
+        'enabled' => env('FLIGHTDECK_CORS_ENABLED', true)
```

Comment:
I don't think either of these should be ENVs. You're unlikely to want to change this, they should just be values that they change in their config file

Reply:
Good point. Updated to remove un-necessary env options.